apis/slo/v1alpha1/nodeslo_types.go

```diff
@@ -25,10 +25,21 @@ import (
 
 // CPUQOS enables cpu qos features.
 type CPUQOS struct {
+	// applied policy for the CPU QoS, default = "groupIdentity"
+	Policy *CPUQOSPolicy `json:"policy,omitempty"`
```

Comment:
Will these two strategies exist at the same time?

Reply:
No, they can not co-exist.